setup.cfg

```diff
@@ -21,6 +21,9 @@ classifiers =
 [options]
 packages = sevsnpmeasure
 python_requires = >=3.7
+install_requires = 
+    cryptography==39.0.1
+    types-cryptography
```

Comment:
Please also add a `requirements.txt` file at the repo's root dir with the following content:

```
cryptography==39.0.1
types-cryptography
```

I think this will allow the CI jobs to install these prior to running the other tasks (`make typecheck` etc).